### ISSUE TITLE
Function call with parenthesis (to group something) do not work

### ISSUE THREAD

**Issue by netei** [eventualbuddha/decaffeinate#96]
```
regex=".*"
"foo".replace (new RegExp(regex)),f
```

```
f (1+1),(2+2)
```

returns : 

```
var regex=".*";
"foo".replace(new RegExp(regex)),f;
```    

and

```
f(1+1),(2+2)); 
```

My guess is that is is an issue in the coffeescript redux parser

**Comment by netei** [comment]
Same thing for 


```
a= new c ([b()])
```

**Comment by eventualbuddha** [comment]
Fixed in v1.41.3.
